sdk/storage/blobs/inc/az_storage_blobs.h

```diff
@@ -53,28 +80,39 @@ typedef struct
   az_span option;
 } az_storage_blobs_blob_upload_options;
 
+/**
+ * @brief Init a client with default options
+ * This is convenient method to create a client with basic settings
+ * Specific options can be modified after initializing the options
```

Comment:
This should not be part of the `@brief` but rather in the `@details` section.



```suggestion
* @details Call this to obtain an initialized #az_storage_blobs_blob_client_options structure that can be modified and passed to #az_storage_blobs_blob_client_init().
```